README.md

```diff
@@ -76,14 +79,29 @@ curl -s -G --data-urlencode "q=+tenant_id:\"tenant-1\" AND title:Japan" localhos
 curl -s -G --data-urlencode "q=+tenant_id:\"tenant-1\" AND title:cup" localhost:8675/try_cloaked_search/_search | jq
 ```
 
+Term queries can also be combined with ORs or ANDs like this:
+
```

Comment:
I wonder if it would be better to combine the search terms into a single subquery?

Reply:
In this case I'm not sure it makes much of a difference. I think it'd be nice to continue to expand this section to show more complex queries.